Launcher/Classes/ViewControllers/SettingsViewController.swift

```diff
@@ -143,6 +153,16 @@ class SettingsViewController: NSViewController {
             self.dismiss(true)
         }
     }
-
+    
+    func fillDictionary(with value: String, for key: String, on fieldName: String) -> [String : Any] {
```

Comment:
Regarding this function - see my comment [here](https://github.com/xing/calabash-launcher/pull/118#discussion_r160111818).

Reply:
This I renamed in another PR